test/lib/playground/serializeRequestBody.test.ts

```
@@ -0,0 +1,61 @@
+  })
+
+  it('passes through FormData, Blob and string bodies', () => {
+    const formData = new FormData()
+    const blob = new Blob(['x'])
+
+    expect(serializeRequestBody(formData)).toBe(formData)
+    expect(serializeRequestBody(blob)).toBe(blob)
+    expect(serializeRequestBody('raw=text')).toBe('raw=text')
+  })
+
+  it('JSON.stringifies object bodies by default', () => {
```

ESLint

[error] `it`s should begin with lowercase

(test/prefer-lowercase-title)